packages/cudd/cudd.0.1.2/opam

```diff
@@ -0,0 +1,81 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Minimal cudd bindings"
+maintainer: [
+  "Matthieu Lemerre <matthieu.lemerre@cea.fr>"
+  "Paul Robert <paul.robert@cea.fr>"
+]
+authors: [
+  "Matthieu Lemerre <matthieu.lemerre@cea.fr>"
+  "Paul Robert <paul.robert@cea.fr>"
+]
+license: "LGPL-2.1-only"
+homepage: "https://git.frama-c.com/pub/codex/cudd.ml"
+bug-reports: "https://git.frama-c.com/pub/codex/cudd.ml/-/issues"
+depends: [
+  "dune" {>= "3.6"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+############################################################################
+##  This file is part of the Codex semantics library.                     ##
+##                                                                        ##
+##  Copyright (C) 2013-2025                                               ##
+##    CEA (Commissariat à l'énergie atomique et aux énergies              ##
+##         alternatives)                                                  ##
+##                                                                        ##
+##  you can redistribute it and/or modify it under the terms of the GNU   ##
+##  Lesser General Public License as published by the Free Software       ##
+##  Foundation, version 2.1.                                              ##
+##                                                                        ##
+##  It is distributed in the hope that it will be useful,                 ##
+##  but WITHOUT ANY WARRANTY# without even the implied warranty of        ##
+##  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+##  GNU Lesser General Public License for more details.                   ##
+##                                                                        ##
+##  See the GNU Lesser General Public License version 2.1                 ##
+##  for more details (enclosed in the file LICENSE).                      ##
+##                                                                        ##
+############################################################################
+
+
+
+dev-repo: "git+https://git.frama-c.com/pub/codex/cudd.ml.git"
+extra-source "cudd/cudd.tar.gz" {
+  src:
+    "https://github.com/ivmai/cudd/archive/refs/tags/cudd-3.0.0.tar.gz"
+  checksum: [
+    "sha256=5fe145041c594689e6e7cf4cd623d5f2b7c36261708be8c9a72aed72cf67acce"
+    "md5=edca9c69528256ca8ae37be9cedef73f"
+  ]
+}
+
+available: [os="linux" & os-family != "windows" & arch != "riscv64"]
+
+depexts: [
+  "bash"
+  "make"
+  "curl"
+  "tar"
+]
```

Comment:
This list of `depexts` is a bit unusual.
Would it make sense to depend on the growing list of `conf-*` packages instead? :thinking: 
This way, enhancements can be made in one place and benefit everyone.
- We already have: `conf-bash` and `conf-libcurl`.
- `make` is already a requirement for the OCaml compiler.
- I'm unsure of `tar` (certainly `opam` already needs some way to extract tar balls)